test/common/quic/envoy_quic_proof_verifier_test.cc

```diff
@@ -192,5 +192,74 @@ VdGXMAjeXhnOnPvmDi5hUz/uvI+Pg6cNmUoCRwSCnK/DazhA
   EXPECT_EQ("Invalid leaf cert, only P-256 ECDSA certificates are supported", error_details);
 }
 
+TEST_F(EnvoyQuicProofVerifierTest, VerifyCertChainFailureNonServerAuthEKU) {
+  root_ca_cert_ = R"(-----BEGIN CERTIFICATE-----
```

Comment:
Do we have documentation somewhere that explains how these certs were generated?

Reply:
Under test/config/integration/certs/ there are bunch of cert configs and certs.sh to generate various kinda cert. I just modified servercert.cfg a bit and run the script.

Commented about where this cert comes from.

Reply:
Perfect, thanks!